source/mesh_deformation/fastscapecc.cc

```diff
@@ -0,0 +1,698 @@
+#include <iostream>
```

Comment:
This file is missing its customary header. Just take it from one of the other files, with the copyright year adjusted.